example_generate.py

```diff
@@ -11,29 +10,23 @@
 # 檔案路徑設定
 #####################################################################
 
-# experiment_no = 1
-experiment_no = 5
+experiment_no = 1
```

Comment:
use `argparse` to get `experiment_no`, wrap whole generation script into a function.